### DISCUSSION TITLE
server: dark mode

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [asciinema/discussions#195] [Ideas]
By using media query [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme).

**Comment by ku1ik** [comment]
Before we add dark theme (if we add at all) I'd like to upgrade the look of the site and CSS foundation it's built on. The current one is based on Bootstrap v4, and I'm thinking about using something more modern, leaner and less opinionated. I don't want to rely on full blown CSS framework, and rather use something minimal.

**Comment by ku1ik** [comment]
I just got rid of Bootstrap and rebuilt the UI using simple, modern CSS, setting up better foundation for UI improvements. I'll tackle dark/light mode the next thing.
